css/css-fonts/variations/at-font-face-font-matching.html

```diff
@@ -79,36 +80,60 @@
         ];
 
         descriptorPriorityCases.forEach(function (testCase) {
-            promise_test(
-                assert => once_fonts_are_ready.then(assert => {
-                    verifyFont("descriptorPriorityTest", testCase.weight, testCase.style, testCase.stretch, testCase.expectedFamily);
-                }),
+            promise_test(() => {
+                return once_fonts_are_ready
+                    .then(() => verifyFont("descriptorPriorityTest", testCase.weight, testCase.style, testCase.stretch, testCase.expectedFamily));
+                },
                 "Descriptor mathcing priority: " + testCase.description
             );
         });
 
         function createFontFaceRules(fontFaceFamily, descriptorName, expectedMatch, unexpectedMatch) {
+            const el = document.createElement("span");
+            el.innerText = "A";
```

Comment:
Instead of just setting el.innerText to a single "A", doing something like this:

            const expect = expectedMatch.split(" ")[0];
            const unexpect = unexpectedMatch.split(" ")[0];
            el.innerHTML = `<span style="${descriptorName}:${expect}">A</span><span style="${descriptorName}:${unexpect}">A</span>`;

seems to help quite a bit, by trying to ensure that loading will be triggered for both faces. (In principle, it doesn't guarantee this, as both the `expect` and `unexpect` property values could still map to the same face.)